tests/TestCase.php

```diff
@@ -154,6 +156,25 @@ protected function assertDeprecated(callable $execution): void
         $this->assertCount(1, $errors);
     }
 
+    protected function createOptionDataProvider(array $options): array
+    {
+        $data = [];
+
+        // Loop through each option and get possible values
```

Comment:
This comment doesn't add much value. Is copilot the new copypasta?

Reply:
The method was more complex originally (it involved a couple of closures), so I decided to add some comments. Then I removed the closure and made this easier but didn't remove the comment.

The only Copilot generated code in this PR is the repetitive `'<option>' => $this->getSomeValues(),` lines 😅